test/integration/gems_test.rb

```diff
@@ -155,22 +155,61 @@ class GemsSystemTest < SystemTest
     assert page.has_no_selector?(".github-btn")
   end
 
-  test "does show version that introduced mfa requirement" do
+  test "shows both blocks if latest AND viewed version require MFA" do
```

Comment:
nit: we could be more explicit with what `blocks` mean in these tests. eg. `both blocks` could be `mfa requirement headers`